tools/macos.py

```diff
@@ -12,6 +13,9 @@ def options(opts):
     opts.Add("macos_sdk_path", "macOS SDK path", "")
     if has_osxcross():
         opts.Add("osxcross_sdk", "OSXCross SDK version", "darwin16")
+    opts.Add(BoolVariable("use_ubsan", "Use LLVM/GCC compiler undefined behavior sanitizer (UBSAN)", False))
```

Comment:
Does macos not support msan and lsan?

Reply:
Yeah not with native compilation tools. google/sanitizers#1026